### DISCUSSION TITLE
Toolbar customization - Insert New Page

### DISCUSSION THREAD

**Discussion by radm70** [xournalpp/xournalpp#3029] [Q&A]
It's possible to change current behaviour of pressing button (in config file as INSERT_NEW_PAGE macro) to
something like INSERT_NEW_PAGE_AT_END ? 
I sometimes insert new page between existing pages instead of at the end, when content is scrolled ;-)

**Comment by Technius** [comment]
Are you using 1.0.20? In the upcoming version 1.1.0, the selected page will be kept in sync when you scroll, so scrolling to the last page and using the insert page button will insert the page at the end.

**Reply by radm70** [reply]
yes, indeed. This is working on the current version too: when I scroll to the last page and insert new one, it appears on the end.
In the main menu -> Journal is option: New Page At the End (or similar, I have translated to Polish version ;-) ). So I asked it is possible to make custom button using this, with macro similar to "INSERT_NEW_PAGE" in config file, something like "NEW_PAGE_AT_END" :-)
